### PR TITLE
adding helper functions to enable getting orders since a given ID or …

### PR DESCRIPTION
**Pull request by danieltmiles**
…since a given order_number

### CODE REVIEW

File: orders.go [hunk 1]
```diff
@@ -208,6 +208,38 @@ func (s *Shopify) GetOrders(creds *Credentials, params url.Values) ([]*Order, er
...
+		return nil, err
+	}
+	return s.GetOrdersSinceId(creds, sinceId, params)
+}
+
+func (s *Shopify) FindOrderIdFromOrderNumber(creds *Credentials, orderNum string) (string, error) {
+	params := url.Values{}
+	params.Add("name", orderNum)
+	params.Add("fields", "id,order_number")
+	uri, err := s.getUri("/admin/orders.json", creds, params)
```
Comment (dan9186) [inline orders.go]: There's a constant available for here.  `OrdersEndpoint`

**Comment by dan9186** [comment]
Aside of the one constant, I see nothing wrong with the code.

I do have more of a philosophical question to follow up with for this.  These functions don't really seem/feel like they're really an implementation of the API, but rather syntactic sugar for our own implementation.  So I guess my question is, do these really belong in the open source library or should we keep it lean and put these in our own use of the library?  I'm fine either way since I imagine someone else could potentially make use of it, but thought I'd ask.

**Comment by dan9186** [comment]
Forgot to add that I'd vote for a self squirrel when you get back since I'll be away eating.

File: orders.go (hunk 1 above)
Comment (danieltmiles) [inline orders.go]: turns out that's actually /admin/orders/%v.json, and I needed orders.json

Comment (danieltmiles) [inline orders.go]: ...correction, it's not, I'll use the constant, nice catch.

**Comment by danieltmiles** [comment]
Yeah, I see the concern. The reason I decided to put it in here is that I found forum posts where other people were having to solve this problem, and after reading that, it felt like a concern that was generalizable and not just sugar for our own usage.

**Comment by danieltmiles** [comment]
self :shipit: since @dan9186 voted to let me do that. :)

**Pull request merged**